### PR TITLE
Add docs for Sentinel 2 L2A.

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Describe the scope of this pull request. Is this a new feature or a bug fix?

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/data/aws-s2-l2a.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.. _sentinel2-l2a-fullscene-block:
+
+Sentinel-2 L2A MSI Full Scenes
+==============================
+Please see the `block details page <https://marketplace.up42.com/block/98c1acfa-c141-4095-b2e6-acf52d110178>`_ for context.
+
+Block type: ``DATA``
+
+This block provides full scenes of Sentinel-2 (A/B)’s multispectral imaging sensor in processing level L-2A in SAFE
+folder structure. The data is collected form `AWS Sentinel 2 bucket <https://registry.opendata.aws/sentinel-2/>`_.
```
Comment (rodrigoalmeida94) [inline source/up42-blocks/data/aws-s2-l2a.rst:14]:
```suggestion
folder structure. The data is collected from `AWS Sentinel 2 bucket <https://registry.opendata.aws/sentinel-2/>`_.
```

File: source/up42-blocks/data/aws-s2-l2a.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+The output format is identical to the unzipped SAFE folder that can be acquired from ESA’s Scihub or
+other DIAS systems. The products are radiometrically and geometrically corrected (including orthorectification).
```
Comment (perusio) [inline source/up42-blocks/data/aws-s2-l2a.rst:16]: other DIAS platforms.

File: source/up42-blocks/data/aws-s2-l2a.rst (hunk 1 above)
Comment (perusio) [inline source/up42-blocks/data/aws-s2-l2a.rst:14]: > in SAFE format. 

No?

File: source/up42-blocks/data/aws-s2-l2a.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Sentinel-2 L2A MSI Full Scenes
+==============================
+Please see the `block details page <https://marketplace.up42.com/block/98c1acfa-c141-4095-b2e6-acf52d110178>`_ for context.
+
+Block type: ``DATA``
+
+This block provides full scenes of Sentinel-2 (A/B)’s multispectral imaging sensor in processing level L-2A in SAFE
+folder structure. The data is collected form `AWS Sentinel 2 bucket <https://registry.opendata.aws/sentinel-2/>`_.
+The output format is identical to the unzipped SAFE folder that can be acquired from ESA’s Scihub or
```
Comment (perusio) [inline source/up42-blocks/data/aws-s2-l2a.rst:15]: uncompressed SAFE format.

File: source/up42-blocks/data/aws-s2-l2a.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    {
+      "aws-s2-l2a:1": {
+        "time": "2018-12-10T16:47:48+00:00/2019-03-10T16:47:49+00:00",
+        "limit": 1,
+        "intersects": {
+          "type": "Polygon",
+          "coordinates": [
+            [
+              [15.106689453125, 51.59754765771458],
+			  [15.809814453125, 51.59754765771458],
```
Comment (perusio) [inline source/up42-blocks/data/aws-s2-l2a.rst:58]: The indentation here is broken.

**Pull request merged**